Framework/Properties/Mantid.properties.template

```diff
@@ -274,3 +274,6 @@ plots.legend.Location = best
 
 # Default legend size
 plots.legend.FontSize = 8
+
+# Default colormap for image plots
+plots.images.Colormap = jet
```

Comment:
Could I ask the reason we are defaulting to Jet? matplotlib switched away from it as a default to viridis as jet has numerous [issues](https://bids.github.io/colormap/). I know people like it but I think it wold be good not to encourage its use.

Reply:
I believe viridis is unavailable on older versions of matplotlib as currently in functions.py there's a try-except that uses jet as the default colormap if importing viridis raises an exception. I figured it was easier to just make jet the default on all OSs but I can add a similar check for setting the default in this file, I'm just not sure where that check should go in the code?

Reply:
Ah okay. I think once we moved to Python 3 we've got viridis available everywhere so can we remove that comment and check `functions.py` now?